resources/assets/coffee/react/_components/comments.coffee

```diff
@@ -27,7 +27,7 @@ export class Comments extends React.PureComponent
 
       div className: classWithModifiers('comments', @props.modifiers),
         div className: 'u-has-anchor u-has-anchor--no-event',
-          div(className: 'fragment-target', id: 'comments')
+          div(id: 'comments')
```

Comment:
`u-has-anchor` and the standalone id element can probably be just removed and the `id` moved to the actual target element.

Reply:
Done.